pkg/apiserver/admission/chain_builder.go

```diff
@@ -210,11 +210,15 @@ func NewAdmissionChains(
 		admissionPluginConfigFilename = tempFile.Name()
 	}
 
-	admissionPluginNames := openshiftAdmissionControlPlugins
-	admissionPluginNames = fixupAdmissionPlugins(admissionPluginNames)
-
-	admissionChain, err := newAdmissionChainFunc(admissionPluginNames, admissionPluginConfigFilename, admissionInitializer, admissionDecorator)
+	admissionPluginNames := sets.NewString(openshiftAdmissionControlPlugins...)
+	for _, pluginName := range DefaultOffPlugins.List() {
+		// turn off some admission plugins by default, unless they are explicitly enabled
+		if _, ok := pluginConfig[pluginName]; !ok {
```

Comment:
this code isn't quite right, but I guess it will do for now.  This doesn't properly handle the disable case I don't think.

Reply:
> his code isn't quite right, but I guess it will do for now. This doesn't properly handle the disable case I don't think.

Actually, go ahead and plumb in arguments to this method for `explicitOn` and `explicitOff` sets.  If it's not in `explicitOn`, then it's off.  It brings us closer to upstream.